Source/dataServer.js

```diff
@@ -11,12 +11,9 @@ var fs       = require('fs'); // TEMP - for saving acquisuite POST data\
 var bodyParser   = require('body-parser');
 var morgan       = require('morgan');
 
-// Connect to and configure the database connection.
-var configDB = require('./config/database.js');
-
 // configuration ===============================================================
 
-mongoose.connect(configDB.url, { useMongoClient: true }); // connect to our database
+mongoose.connect(z, { useMongoClient: true }); // connect to our database
```

Comment:
Where is Z defined? Is this just the URL to the database?

Reply:
fixed. should be ENV VAR